modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -260,6 +270,33 @@ const Modal = ({
 	);
 };
 
+const confirm = ({message, onConfirm}) => {
+	openModal({
+		bodyHTML: message,
+		buttons: [
+			{
+				displayType: 'secondary',
+				label: Liferay.Language.get('cancel'),
+				type: 'cancel',
+			},
+			{
+				label: Liferay.Language.get('ok'),
+				onClick: ({processClose}) => {
+					processClose();
+
+					onConfirm(true);
+				},
+				otherProps: {
+					autoFocus: true,
+				},
```

Comment:
Do we need the `otherProps` prop here? Why not just `autoFocus` prop? `otherProps` implies that we are supporting any prop on a button. For this, we would not need the explicit `otherProps` prop. In the implementation, we would deconstruct the `button` into explicit props and pass the remaining to the underlying component. But, I don't think this is the case here, IIUC the intent is just to expand API with a simple prop.

Reply:
> otherProps implies that we are supporting any prop on a button

The main goal it is, I think can be easier for people that have used Clay components to see and try to pass Clay props there. I think converting these commands to TS and sharing the same interface from ClayButton would be nice to really tell the user: Here, you can pass whatever you want based on Clay Button props. What do you think? Also calling @matuzalemsteles @julien  because I think it can help the extensibility of these commands

Reply:
If you wish to support any prop on a button, I think it's fine. It is not required for the task at hand, but it makes sense as an API  :+1:

But then, to quote myself:
>  we would not need the explicit otherProps prop. In the implementation, we would deconstruct the button into explicit props and pass the remaining to the underlying component

Reply:
> we would not need the explicit otherProps prop. In the implementation, we would deconstruct the button into explicit props and pass the remaining to the underlying component

Yes, for sure. I changed to not needing having otherProps property anymore based on: https://github.com/liferay-frontend/liferay-portal/pull/1990#discussion_r827914270

Thanks Marko 💖

Reply:
I don't see those changes in PR :mag:

Reply:
See commits tab again, you'll get a surprise. Thanks for the eagle eyes Marko 🦅